website/docs/r/virtual_desktop_application.html.markdown

```diff
@@ -0,0 +1,114 @@
+---
+
+# virtual_desktop_application
+
+Manages a Virtual Desktop Application.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "rg-example-virtualdesktop"
+  location = "West Europe"
+}
+
+resource "azurerm_virtual_desktop_host_pool" "pooledbreadthfirst" {
+  name                = "pooledbreadthfirst"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  type               = "Pooled"
+  load_balancer_type = "BreadthFirst"
+}
+
+resource "azurerm_virtual_desktop_host_pool" "personalautomatic" {
+  name                = "personalautomatic"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  type                             = "Personal"
+  personal_desktop_assignment_type = "Automatic"
+}
+
+resource "azurerm_virtual_desktop_application_group" "remoteapp" {
+  name                = "acctag"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  type          = "RemoteApp"
+  host_pool_id  = azurerm_virtual_desktop_host_pool.pooledbreadthfirst.id
+  friendly_name = "TestAppGroup"
+  description   = "Acceptance Test: An application group"
+}
+
+resource "azurerm_virtual_desktop_application" "chrome" {
+  name                   = "googlechrome"
+	application_group_id   = azurerm_virtual_desktop_application_group.remoteapp.id
+	friendly_name 		     = "Google Chrome"
+	description 		       = "Chromium based web browser"
+	file_path              = "C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe"
+	command_line_setting   = "DoNotAllow"
+	command_line_arguments = "--incognito"
+	show_in_portal         = false
+	icon_path			         = "C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe"
+	icon_index             = 0
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Virtual Desktop Application. Changing the name forces a new resource to be created.
+
+* `application_group_id` - (Required) Resource ID for a Virtual Desktop Application Group to associate with the
+    Virtual Desktop Application. Changing the ID forces a new resource to be created.
+
+* `friendly_name` - (Optional) Option to set a friendly name for the Virtual Desktop Application.
+
+* `description` - (Optional) Option to set a description for the Virtual Desktop Application.
+
+* `file_path` - (Required) The file path location of the app on the Virtual Desktop OS.
+
+* `command_line_setting` - (Required) Specifies whether this published application can be launched with command line arguments provided by the client, command line arguments specified at publish time, or no command line arguments at all. Possible values include: `DoNotAllow`, `Allow`, `Require`.
```

Comment:
i think we can just use this to be more consistant
```suggestion
* `command_line_argument_policy` - (Required) Specifies whether this published application can be launched with command line arguments provided by the client, command line arguments specified at publish time, or no command line arguments at all. Possible values include: `DoNotAllow`, `Allow`, `Require`.
```

so the tf becomes `command_line_argument_policy = allow`